astropy/io/fits/tests/test_image.py

```diff
@@ -253,7 +248,7 @@ def test_io_manipulation(self):
         hdu2 = fits.ImageHDU(header=r[1].header, data=np.array([1, 2],
                              dtype='int32'))
 
-        assert (str(hdu2.header.ascard[1:5]) ==
+        assert ('\n'.join(str(x) for x in hdu2.header.cards[1:5]) ==
```

Comment:
Nope, `ascard` was returning a `CardList` which had directly a string representation. `.cards` now return a list of `Cards`, so one need to get and join the string representation of each card. 
Another option could be `str(hdu2.header[1:5]))` but not fully equivalent as it includes the `END` keyword.